src/App.js

```diff
@@ -18,12 +19,18 @@ function App(props) {
   const { classes } = props;
   return (
     <div className={classes.root}>
-      <Router>
-      <Switch>
-        <Route path="/" component={Home} exact />
-        <Route path="/stream" component={StreamView} exact />
-      </Switch>
-      </Router>
+    <Router>
+    <Switch>
+      <Route path="/" component={Home} exact />
+      <Route path="/stream" component={StreamView} exact />
+      <Route path="/hosthome" component={HostHome} exact />
+      <Route path="/hostdashboard" component={HostDashboard} exact />
+      <Route path="/hostdashboard/Overview" component={HostDashboard} exact />
```

Comment:
Try configuring child routes as mentioned in the docs (https://reactrouter.com/web/example/route-config)